mobile/native/src/ln_dlc/node.rs

```diff
@@ -77,8 +78,8 @@ impl Node {
         })
     }
 
-    pub async fn get_wallet_histories(&self) -> Result<WalletHistories> {
-        let on_chain = self.inner.get_on_chain_history().await?;
+    pub fn get_wallet_histories(&self) -> Result<WalletHistories> {
```

Comment:
🔧 Caller of this no longer needs to be `async` because of this. It would be great if we could update the callers of `keep_wallet_balance_and_history_up_to_date` too to use `spawn_blocking` where needed.

Reply:
Addressed here - https://github.com/get10101/10101/pull/950/commits/8fc8096e9a69a7415c9afe4e0b41dd0fabffc7b1

Please have a look if this is what you intended.